<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddContentDialog</class>
 <widget class="QDialog" name="AddContentDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DownZemAll - Add Web Page Content</string>
  </property>
  <property name="styleSheet">
   <string notr="true">/* QGroupBox exposes the title as QString (even if internally its probably a QLabel) */
QGroupBox {
	font-weight: bold;
}</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainVerticalLayout" stretch="1,0,0,0,0">
     <property name="spacing">
      <number>6</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="topMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>9</number>
     </property>
     <property name="bottomMargin">
      <number>9</number>
     </property>
     <item>
      <widget class="QStackedWidget" name="stackedWidget">
       <widget class="QWidget" name="mainPage">
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <property name="bottomMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="LinkWidget" name="linkWidget" native="true"/>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="progressPage">
        <layout class="QGridLayout" name="gridLayout" rowstretch="1,0,0,1" columnstretch="1,0,1">
         <item row="2" column="2">
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="1">
          <widget class="QProgressBar" name="progressBar">
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="textVisible">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="2" column="0">
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="3" column="1">
          <spacer name="verticalSpacer_2">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item row="1" column="1">
          <widget class="QLabel" name="progressLabel">
           <property name="text">
            <string>Collecting...</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="1,0">
       <property name="spacing">
        <number>6</number>
       </property>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Save files in:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="PathWidget" name="pathWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>10</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Default Mask:</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="MaskWidget" name="maskWidget" native="true">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>10</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="FilterWidget" name="filterWidget" native="true">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>50</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QWidget" name="fastFilteringWidget" native="true"/>
     </item>
     <item>
      <layout class="QHBoxLayout" name="buttonBox" stretch="1,0,0,0">
       <property name="spacing">
        <number>9</number>
       </property>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="startButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>&amp;Start!</string>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addPausedButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Add &amp;paused</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="cancelButton">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>30</height>
          </size>
         </property>
         <property name="text">
          <string>Cancel</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout" stretch="1,0,0">
     <property name="spacing">
      <number>0</number>
     </property>
     <property name="leftMargin">
      <number>9</number>
     </property>
     <property name="rightMargin">
      <number>29</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="tipLabel">
       <property name="text">
        <string>tipLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QToolButton" name="preferenceButton">
       <property name="toolTip">
        <string>Preferences</string>
       </property>
       <property name="text">
        <string>Preferences</string>
       </property>
       <property name="icon">
        <iconset resource="../resources.qrc">
         <normaloff>:/icons/menu/button_preference_16x16.png</normaloff>:/icons/menu/button_preference_16x16.png</iconset>
       </property>
       <property name="toolButtonStyle">
        <enum>Qt::ToolButtonTextBesideIcon</enum>
       </property>
       <property name="autoRaise">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>PathWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/PathWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>MaskWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/MaskWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>LinkWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/LinkWidget</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>FilterWidget</class>
   <extends>QWidget</extends>
   <header>Widgets/FilterWidget</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>startButton</sender>
   <signal>released()</signal>
   <receiver>AddContentDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>565</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>79</x>
     <y>571</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addPausedButton</sender>
   <signal>released()</signal>
   <receiver>AddContentDialog</receiver>
   <slot>acceptPaused()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>685</x>
     <y>548</y>
    </hint>
    <hint type="destinationlabel">
     <x>644</x>
     <y>569</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>cancelButton</sender>
   <signal>released()</signal>
   <receiver>AddContentDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>764</x>
     <y>547</y>
    </hint>
    <hint type="destinationlabel">
     <x>762</x>
     <y>567</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>acceptPaused()</slot>
 </slots>
</ui>
